src/ChoosePlatform.tsx

```diff
@@ -28,6 +41,13 @@ export default class ChoosePlatform extends Component<{}, {}> {
   }
 
   render() {
+    const allowed_platforms = Artist_platform
+      .filter(platform => this.props.event[platformAllowances[platform]]);
+
+    if (allowed_platforms.length == 1) {
```

Comment:
Shouldn't this be generalized to handle all the cases when the platforms are any length

Reply:
It is - if there is more than one allowed platform, then the selection menu is rendered below. The redirect need only happen if there's exactly one choice of platform, no?